mantis-common/src/test/java/io/mantisrx/common/compression/CompressionUtilsTest.java

```diff
@@ -45,7 +46,7 @@
     }
 
     @Test public void shouldTokenizeWithEventsContainingPartialDelimiterMatchesWithCustomDelimiter() {
-        String delimiter = UUID.randomUUID().toString();
+        String delimiter = "a04f0418-bdff-4f53-af7d-9f5a093b9d65";
```

Comment:
I'm curious why the fixed value here?

Reply:
This was to get around a bug fixed by #154